interfaces/mount/entry.go

```diff
@@ -141,3 +144,49 @@ func ParseEntry(s string) (Entry, error) {
 	e.CheckPassNumber = cpn
 	return e, nil
 }
+
+// LoadFSTab reads and parses an fstab-like file.
+//
+// The supported format is described by fstab(5).
+func LoadFSTab(reader io.Reader) ([]Entry, error) {
```

Comment:
I was trying to follow the acronym (file system table) but I have no strong opinions about it. We can rename this to `{Load,Save}Fstab` if you want.